yarn.lock

```diff
@@ -282,9 +282,10 @@
     esutils "^2.0.2"
     js-tokens "^4.0.0"
 
-"@babel/parser@7.2.0":
-  version "7.2.0"
-  resolved "https://registry.yarnpkg.com/@babel/parser/-/parser-7.2.0.tgz#02d01dbc330b6cbf36b76ac93c50752c69027065"
+"@babel/parser@7.6.0", "@babel/parser@^7.6.0":
```

Comment:
We should do the upgrade in a different PR and fix the few tests that are breaking as a result.